src/wp-includes/option.php

```diff
@@ -2144,20 +2144,45 @@ function update_network_option( $network_id, $option, $value ) {
 	 */
 	$value = apply_filters( "pre_update_site_option_{$option}", $value, $old_value, $option, $network_id );
 
+	/*
```

Comment:
Can we get rid of `$old_value = get_network_option( $network_id, $option, false );` now?